pandunia-lekse.csv

```diff
@@ -2897,7 +2897,7 @@ xatani||||devilish (satanic)||satánico||||||||||||||pirullinen|diabelski, szata
 xataniste||||Satanist||satanista||||||||||||||satanisti|satanista|
 xatanistia||||Satanism (devil worship)||satanismo||||||||||||||satanismi (paholaisenpalvonta)|satanizm|
 xatranje|||hin:शतरंज (śatrañj), urd:(śatrañj), fas:(šatranj), tur:satranç, ara:(šaṭranj), swa:sataranji, tel:చతురంగము (caturaṅgamu), tam:சதுரங்கம் (caturaṅkam), may:catur, khm:ចត្រង្គ (caʾtrɑng), por:xadrez|chess||ajedrez||||||||शतरंज||catur|sataranji||ŝako|šakki|szachy|
-xaw|||zho:少 (shǎo), yue:少 (siu2), jpn:少 (shō), kor:소 (so)|little (few, not many)|peu|pocos|poucos|мало||少||||कम|||||malmulte|vähän|niewiele (mało)|
+xaw (xawi)|||zho:少 (shǎo), yue:少 (siu2), jpn:少 (shō), kor:소 (so)|little (few, not many)|peu|pocos|poucos|мало||少||||कम|||||malmulte|vähän|niewiele (mało)|
```

Comment:
Here is another another word where the vowel ending ruins the beauty of it. :( I think we should find another word that would sound good with -i.

Reply:
That's fair, though "xaw" was a very good word. We could use "poki" from Latin? I also think it would be fair if simple adjectives like "pol" and "xaw" were among those few roots that don't need endings.